torch/csrc/jit/script/module.h

```diff
@@ -150,7 +150,7 @@ struct Method {
     return *this;
   }
 
-  const FunctionSchema& getSchema() const;
```

Comment:
It seems that `module.h` gets referenced in other header files, so the function is exported more than once, hence the linker error. So the probable solutions are
1. Move the implementation of this function here and remove TORCH_API
2. Refactor the code and eliminate the reference to this class(function)

Reply:
Thanks a lot @peterjc123, (1) solved it!